FogCreek/Glitch-Community
# Polyester wallet: bookmark button should update when we add a project to My stuff from popover

## Links
* Remix link: https://polyester-wallet.glitch.me

## GIF/Screenshots:

![](http://g.recordit.co/hORVusECHX.gif)

## Changes:
* wraps the `addProjectToCollection` function in project-item and project page to set the bookmark to filled if relevant. 

## How To Test:
* enable my stuff
* add a project to my stuff via the add to project collection popover on a collection page, user page, team page, search page or project page.
* see the button light up

## Feedback I'm looking for:
- i did start going down the path of putting this data into the project provider in state/project.js instead, but it was feeling like a lot of refactoring and maybe not worth it? but maybe you feel like it is worth it and maybe I just needed more coffee. I'm open to suggestions here.
